<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notes2</class>
 <widget class="QMainWindow" name="Notes2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notes2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="gbSearch">
        <property name="title">
         <string>Search</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <property name="spacing">
          <number>3</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QCheckBox" name="chbText">
               <property name="text">
                <string>Text</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chbCaption">
               <property name="text">
                <string>Caption</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLineEdit" name="leSearch"/>
             </item>
             <item>
              <widget class="QPushButton" name="pbSearchClear">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QListWidget" name="lwNotes">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTextBrowser" name="tbNoteText">
            <property name="font">
             <font>
              <pointsize>50</pointsize>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <property name="midLineWidth">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="lcTags">
              <property name="text">
               <string>Tags:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lTags">
              <property name="text">
               <string>na</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="lcCreation">
              <property name="text">
               <string>Cr:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lCr">
              <property name="text">
               <string>na</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lCrAgo">
              <property name="text">
               <string>na</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="lcModification">
              <property name="text">
               <string>Md:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lMd">
              <property name="text">
               <string>na</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lMdAgo">
              <property name="text">
               <string>na</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pbNewNote">
          <property name="text">
           <string>New note</string>
          </property>
          <property name="icon">
           <iconset resource="notes2.qrc">
            <normaloff>:/img/img/Add.png</normaloff>:/img/img/Add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbXAction">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit note</string>
          </property>
          <property name="icon">
           <iconset resource="notes2.qrc">
            <normaloff>:/img/img/Modify.png</normaloff>:/img/img/Modify.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbRemoveNote">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove note</string>
          </property>
          <property name="icon">
           <iconset resource="notes2.qrc">
            <normaloff>:/img/img/Remove.png</normaloff>:/img/img/Remove.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb">
          <property name="text">
           <string>toggle filter</string>
          </property>
          <property name="icon">
           <iconset resource="notes2.qrc">
            <normaloff>:/img/img/Search.png</normaloff>:/img/img/Search.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbExit">
          <property name="text">
           <string>Exit</string>
          </property>
          <property name="icon">
           <iconset resource="notes2.qrc">
            <normaloff>:/img/img/Exit.png</normaloff>:/img/img/Exit.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_base"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuNotes">
    <property name="title">
     <string>Notes</string>
    </property>
    <addaction name="actionAdd_note"/>
    <addaction name="actionEdit_note"/>
    <addaction name="actionRemove_note"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_program"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSearch"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuNotes"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_base"/>
   <addaction name="separator"/>
   <addaction name="actionSearch"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_note"/>
   <addaction name="actionEdit_note"/>
   <addaction name="actionRemove_note"/>
   <addaction name="separator"/>
   <addaction name="actionAbout_program"/>
   <addaction name="actionAbout_Qt"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="notes2.qrc">
     <normaloff>:/img/img/Exit.png</normaloff>:/img/img/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit application</string>
   </property>
   <property name="statusTip">
    <string>Exit application</string>
   </property>
  </action>
  <action name="actionAdd_note">
   <property name="icon">
    <iconset resource="notes2.qrc">
     <normaloff>:/img/img/Add.png</normaloff>:/img/img/Add.png</iconset>
   </property>
   <property name="text">
    <string>Add note</string>
   </property>
   <property name="toolTip">
    <string>Add new note</string>
   </property>
  </action>
  <action name="actionRemove_note">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notes2.qrc">
     <normaloff>:/img/img/Remove.png</normaloff>:/img/img/Remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove note</string>
   </property>
   <property name="toolTip">
    <string>Remove current note</string>
   </property>
  </action>
  <action name="actionAbout_program">
   <property name="icon">
    <iconset resource="notes2.qrc">
     <normaloff>:/img/img/About.png</normaloff>:/img/img/About.png</iconset>
   </property>
   <property name="text">
    <string>About program</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="notes2.qrc">
     <normaloff>:/img/img/Qt.png</normaloff>:/img/img/Qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionOpen_base">
   <property name="icon">
    <iconset resource="notes2.qrc">
     <normaloff>:/img/img/Open.png</normaloff>:/img/img/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open base</string>
   </property>
   <property name="toolTip">
    <string>Open existing base</string>
   </property>
  </action>
  <action name="actionEdit_note">
   <property name="icon">
    <iconset resource="notes2.qrc">
     <normaloff>:/img/img/Modify.png</normaloff>:/img/img/Modify.png</iconset>
   </property>
   <property name="text">
    <string>Edit note</string>
   </property>
   <property name="toolTip">
    <string>Edit current note</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/Search.png</normaloff>
     <disabledoff>:/img/img/Search.png</disabledoff>
     <disabledon>:/img/img/Search.png</disabledon>img/Search.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="toolTip">
    <string>Seach text in notes</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="notes2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>actionExit</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbNewNote</sender>
   <signal>clicked()</signal>
   <receiver>actionAdd_note</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
